### PR TITLE
Development

### PR DESCRIPTION
**Pull request by asztalosors**
Preparation for the release of 1.3.0

The release contains:
* bug fixes
* neutral - beam atomic interactions
* comprehensive unit testing
* noise modelling
* refactoring of utility branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,17 @@
 # renate-od
 RENATE Open Diagnostics (RENATE-OD)
 
-Responsible developer: Dr. Gergo Pokol, www.reak.bme.hu/pokol
+Responsible developers:
+* [Dr. Gergo Pokol](www.reak.bme.hu/pokol)
+* [Ors Asztalos](https://github.com/asztalosors)
+
```
Comment (gergopokol) [inline README.md:7]: Peter Balazs and Matyas Aradi to be added

File: utility/manage.py [hunk 2]
```diff
@@ -0,0 +1,236 @@
+import os
```
Comment (gergopokol) [inline utility/manage.py:1]: This should be run and tested.

**Review by gergopokol — CHANGES_REQUESTED** [review]
Manage routines to be run in a dedicated branch. 
Minor changes requested.

File: RENATE-OD.info [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<xml lang="en">
+    <head>
+        <code_name>RENATE Open Diagnostics</code_name>
+        <code_abbreviation>renate-od</code_abbreviation>
+    </head>
+    <body>
+        <code_license>LGPL-3.0</code_license>
+        <code_version>1.2.0</code_version>
```
[the remark this answers is not in the document]
Comment (asztalosors) [reply RENATE-OD.info]: Done

File: utility/manage.py (hunk 2 above)
Comment (asztalosors) [reply utility/manage.py:1]: Done

File: README.md (hunk 1 above)
Comment (asztalosors) [reply README.md:7]: Done

**Review by Balazs-Peter — APPROVED** [review]
Everything seems to be ready for release.

**Review by gergopokol — APPROVED** [review]
Checked. Approved.

**Pull request merged**